tests/e2e/iso_test.go

```diff
@@ -44,6 +44,17 @@ var _ = Describe("ISO image generation", Label("iso"), func() {
 			_, err = os.Stat(filepath.Join(tempDir, "build/build/kairos.iso"))
 			Expect(err).ToNot(HaveOccurred())
 		})
+		It("fails if cloud config is empty", func() {
+			err := WriteConfig("", tempDir)
+			Expect(err).ToNot(HaveOccurred())
+
+			out, err := RunAurora(fmt.Sprintf(`--set container_image=quay.io/kairos/core-rockylinux:latest \
+			--set "disable_http_server=true" \
+			--set "disable_netboot=true" \
+			--cloud-config /config.yaml \
+			--set "state_dir=/tmp/auroraboot"`), tempDir)
+			Expect(err).To(HaveOccurred(), out)
```

Comment:
There is a `MatchError` matcher we can use, to make sure we are getting the expected error and not another one.
```suggestion
			Expect(err).To(MatchError("cloud config set but contents are empty"), out)
```

Reply:
I thought catching exact error messages was frowned upon, as the text can change over time?

Reply:
it better still work now :P